crates/nu-command/src/filters/reduce.rs

```diff
@@ -50,6 +50,11 @@ impl Command for Reduce {
                 description: "Sum values of a list (same as 'math sum')",
                 result: Some(Value::test_int(10)),
             },
+            Example {
+                example: "[ 1 2 3 4 ] | reduce {|it, acc| $acc - $it }",
```

Comment:
I think maybe an example where the left-associativity matters would be better, e.g. by using `--fold`

```nushell
[[foo baz] [baz quux]] | reduce --fold "foobar" {|it, acc| $acc | str replace $it.0 $it.1}
```

It's slightly more complicated but the result would be different if it were right-associative

Reply:
i'd keep the new example and just add devyn's too (with explanation). both would be helpful.